tests/pytest_tests/unit_tests/test_ssl.py

```diff
@@ -84,6 +84,7 @@ def disable_ssl_context():
         reset()
 
 
+@pytest.skip(reason="started failing 2023-06-16 for not-yet-diagnosed reasons")
```

Comment:
Can we use skipif and limit this to python3.7?

Reply:
Agreed